cluster/images/kubelet/Makefile

```diff
@@ -1,21 +1,21 @@
-# build the kubelet image.
+# Build the kubelet image.
+#
+# Usage:
+#   VERSION=v1.1.2 [REGISTRY="gcr.io/google_containers"] make build
 
-# TODO: figure out the best base image
-# TODO: figure out how to best source the nsenter binary, if necessary
-# TODO: figure out how to make a single source of version info for these
-#       images
-VERSION=v0.16
```

Comment:
I think that Makefiles for `hyperkube`, `kubelet` and `kubelet-checker` are similar. What other files do you mean?